### ISSUE TITLE
Seems Ebay has changed the format to stop this working?

### ISSUE THREAD

**Issue by davidmolony** [eugenmihailescu/ebay-purchase-report#21]
![image](https://user-images.githubusercontent.com/16707015/137807862-9c20d3c2-7d99-44ce-b32a-8f2671e7883b.png)

It seems that changes to the ebay layout has resulted in the generate Quick Report button disappearing.

Was a very useful program to me, once a year for accounts...

**Comment by outfielder** [comment]
Yes, seems to be the case with me too. I have not got that option anymore.

**Comment by cattyrunes** [comment]
Same here, quite unfortunate.

**Comment by 1AngelaTC** [comment]
I am crushed that this does not work any more.  What a great tool this was.

**Comment by arminlinder** [comment]
Yes, sadly, this tool is dead. Tested on Firefox and Chrome.

**Comment by fogoat** [comment]
You can request up to last 12 months. Go to Account > Request your eBay Data ... or try here: https://www.sarweb.ebay.com/sar
